### ISSUE TITLE
Self-closing title not reported as an error when validating EPUB 3.2

### ISSUE THREAD

**Issue by JayPanoz** [w3c/epubcheck#1093]
Given

```
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops">
<head>
<title/>
</head>
<body>
</body>
</html>
```

validator.nu (XHTML preset) reports an error: 

```
Error: Element title must not be empty.
From line 6, column 1; to line 6, column 8
">↩<head>↩<title/>↩</hea
```

But epubcheck 4.2.2 doesn’t. 

Reduced test case: 
[selfclosingtitle.zip](https://github.com/w3c/epubcheck/files/3979248/selfclosingtitle.zip)

**Comment by rdeltour** [comment]
Thanks for the report @JayPanoz!

This check is implemented in the Nu HTML Checker as Java code ([`TextContentChecker.java`](https://github.com/validator/validator/blob/18.11.5/src/nu/validator/checker/TextContentChecker.java#L130), and until we don’t fully integrate this checker we’re only using its schemas.

For now, we can re-implement this check as a Schematron rule.
